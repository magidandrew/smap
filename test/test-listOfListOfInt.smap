int main (){
    int i = 0;
    list list int x = [[0,1],[5,4,3,2,1]];
    println(x[0][0]);
    println(x[1][1]);
    for(i; i< 5; i= i+1){
       print(x[1][i]);
    }
    println(x[0]);
    i = 7;
    return (0);
}