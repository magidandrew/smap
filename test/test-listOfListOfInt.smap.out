0

4

5
4
3
2
1
[0, 1]